APE0.rst

```diff
@@ -0,0 +1,360 @@
+responsibilities of the Ombudsperson, should the need arise.
+
+The Voting Members
+------------------
+
+Role
+^^^^
+The Voting Members are the group of trusted individuals who operate the Astropy
+Project on behalf of the community. They have authority over the Astropy
+Project’s technical resources, including the Astropy Project website itself, the
+Astropy GitHub organization and repositories, the issue tracker, and all Astropy
+communication channels. In practice, much of this authority is passed to the
+Coordination Committee via the voting processes described in this Charter.
+
+They also assume many roles required to achieve the Project's goals, especially
+those that require a high level of trust. Collectively, they make decisions that
+shape the future of the Project.
+
+Voting Members are expected to act as role models for the community and
+custodians of the Project, on behalf of the community and all those who rely on
+Astropy. They will act as representatives of the Astropy Project, where
+necessary, including in online discussions or at official Astropy events.
+
+Powers
+^^^^^^
+Voting Members may participate in formal votes on:
+* adding new Voting Members,
+* removing Voting Members,
+* electing Coordination Committee members,
+* removing Coordination Committee members,
+* appointment or removal of the Ombudsperson,
+* changes to this Charter,
+* other matters for which the Coordination Committee believes a vote is
+  appropriate.
+
+The mechanism, timeline, and criteria for a decisive vote are specified in the
+respective sections of this document in the first six cases and by the
+Coordination Committee in the last case.
+
+Membership
+^^^^^^^^^^
+Voting Members of the Astropy Project must demonstrate:
+* a good grasp of the philosophy of the Astropy Project,
+* a solid track record of being constructive and helpful,
+* significant contributions to the Project's goals, in any form,
+* a willingness to dedicate time to improving the Project,
+* a willingness to recruit, train, and nominate new team members.
+
+Voting Membership acknowledges sustained and valuable efforts that align well
+with the philosophy and the goals of the Astropy Project.
+
+Initial Membership
+^^^^^^^^^^^^^^^^^^
+Anyone who satisfies any of the following criteria at the date of acceptance of
+this APE will be invited to be in the first group of Voting Members:
+
+* has a named role in the project and has participated in an Astropy
+  coordination meeting in the last two years,
+* has a named role in the project and has regularly participated in Astropy
+  telecons (e.g., co-working telecon, spectroscopy sprint, Astropy Learn
+  telecon) in the last two years,
+* has commit rights to at least one repository in the astropy GitHub
+  organization and has actively used those commit rights in the last two years.
+
+Add New Voting Members
+^^^^^^^^^^^^^^^^^^^^^^
+Anyone can apply to become a Voting Member by providing evidence of meeting the
```

Comment:
```suggestion
Anyone can become a Voting Member by providing evidence of meeting the
```
"apply" sounds like "self-application" which is not what the next sentence says.